concordia/admin/__init__.py

```diff
@@ -46,6 +49,25 @@
 )
 
 
+class ProjectListFilter(MultipleChoiceListFilter):
+    title = "Project"
+
+    def lookups(self, request, model_admin):
+        qs = Project.objects.all()
```

Comment:
This block would be a little more efficient as something like `choices = Project.objects.values_list("pk", "title")`